cli/channels.go

```diff
@@ -37,17 +37,28 @@ var cmdChannels = []cobra.Command{
 		},
 	},
 	{
-		Use:   "get [all | <channel_id>] <user_auth_token>",
+		Use:   "get [all | name <channel_name> | metadata <metadata_json_string> | <channel_id>] <user_auth_token>",
```

Comment:
I'd use `name` and `metadata` as flags like we did for other filters like `limit` and `offset`

Reply:
Like `get --name <name_valu>`? This makes sense.

Reply:
Yes `--name` and `--metadata`